packages/protocol-contracts/contracts/ZetaConnector.base.sol

```diff
@@ -60,6 +65,12 @@ contract ZetaConnectorBase is ConnectorErrors, Pausable {
         zetaToken = zetaToken_;
         tssAddress = tssAddress_;
         tssAddressUpdater = tssAddressUpdater_;
+        pauserAddress = pauserAddress_;
```

Comment:
pauser can be zero? if not add zero validation

Reply:
👌